tests/test-model-load-cancel.cpp

```diff
@@ -0,0 +1,17 @@
+#include "llama.h"
+
+#include <cstdlib>
+#include <tuple>
+
+int main(void) {
+    llama_backend_init(false);
+    auto params = llama_model_params{};
+    params.use_mmap = false;
+    params.progress_callback = [](float progress, void * ctx){
+        std::ignore = ctx;
```

Comment:
According to cppreference, "the behavior of std::ignore outside of std::tie is not formally specified."